x-pack/plugins/ml/server/models/data_recognizer/modules/security_linux/ml/v3_linux_rare_metadata_process.json

```diff
@@ -40,6 +40,8 @@
       "author": "@randomuserid/Elastic",
       "version": "3",
       "updated_date": "5/16/2022"
-  },
-    "created_by": "ml-module-security-linux"  }
+    },
+    "created_by": "ml-module-security-linux",
```

Comment:
Some of the jobs in this module use a `created_by` of `ml-module-security-linux` and some use `ml-module-security-linux-v3`. It's been like this for a while - is this difference deliberate?

Reply:
This has been around for a while, but I can make updates to keep things consistent. And if both have been working so far, I don't think changing to either one will break things. I'm leaning towards making that field version independent.